scene/gui/file_dialog.cpp

```diff
@@ -863,9 +863,7 @@ void FileDialog::update_file_list() {
 			DirInfo di;
 			di.name = dir_name;
 			di.bundle = bundle;
-			if (file_sort == FileSortOption::MODIFIED_TIME || file_sort == FileSortOption::MODIFIED_TIME_REVERSE) {
-				di.modified_time = FileAccess::get_modified_time(base_dir.path_join(dir_name));
-			}
+			di.modified_time = FileAccess::get_modified_time(base_dir.path_join(dir_name));
```

Comment:
What is this change for? Modified time is not needed for other sorting modes, this condition was supposed to avoid reading it unnecessarily.